spec/unit/default_repo.rb

```diff
@@ -0,0 +1,29 @@
+require 'spec_helper'
```

Comment:
Spec files must be suffixed by `_spec.rb`
Please move this file into `spec/unit/resources/repo_spec.rb`